pkgs/applications/science/misc/root/default.nix

```diff
@@ -176,7 +176,8 @@ stdenv.mkDerivation rec {
   postInstall = ''
     for prog in rootbrowse rootcp rooteventselector rootls rootmkdir rootmv rootprint rootrm rootslimtree; do
       wrapProgram "$out/bin/$prog" \
-        --prefix PYTHONPATH : "$out/lib"
+        --set PYTHONPATH "$out/lib" \
+        --set LD_LIBRARY_PATH "$out/lib"
```

Comment:
```suggestion
        --set ${lib.optionalString stdenv.isDarwin "DY"}LD_LIBRARY_PATH "$out/lib"
```

Reply:
I think for newer macOS the `DYLD_LIBRARY_PATH` setting is ineffective by default? But I guess there's no harm to do this.